docs/docs/spark-getting-started.md

```diff
@@ -41,20 +41,27 @@ spark-shell --packages org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:{{ iceb
 
 ### Adding catalogs
 
-Iceberg comes with [catalogs](spark-configuration.md#catalogs) that enable SQL commands to manage tables and load them by name. Catalogs are configured using properties under `spark.sql.catalog.(catalog_name)`.
+Apache Iceberg provides several [catalog](spark-configuration.md#catalogs) implementations to manage tables and enable SQL operations. 
```

Comment:
Before linking to the Spark configuration, how about linking to the Catalogs concept page (https://iceberg.apache.org/concepts/catalog/) to educate the reader on the importance of a catalog.